code/__DEFINES/is_helpers.dm

```diff
@@ -69,6 +69,7 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 #define isethereal(A) (is_species(A, /datum/species/ethereal))
 #define isvampire(A) (is_species(A,/datum/species/vampire))
 #define isdullahan(A) (is_species(A, /datum/species/dullahan))
+#define isdwarf(A) (is_species(A, /datum/species/human/dwarf))
```

Comment:
one issue; this means ishumanbasic will return true if you check for dwarves because it runs an istype

Reply:
catgirls have specific checks dont they